<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMMainWindow</class>
 <widget class="QMainWindow" name="QMMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QualificationMatrix</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rcs/icons.qrc">
    <normaloff>:/icons/icons/qm.svg</normaloff>:/icons/icons/qm.svg</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMInfoLabel" name="laInfo">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="actManageDatabase"/>
    <addaction name="actCloseDatabase"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hi&amp;lfe</string>
    </property>
    <addaction name="actAboutQt"/>
    <addaction name="actAbout"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Ex&amp;tras</string>
    </property>
    <addaction name="actSettings"/>
   </widget>
   <widget class="QMenu" name="menuNachweise">
    <property name="title">
     <string>&amp;Nachweise</string>
    </property>
    <addaction name="actIntegrityCheck"/>
    <addaction name="actManageCertificate"/>
    <addaction name="actCreateSigningList"/>
   </widget>
   <addaction name="menu_Datei"/>
   <addaction name="menuNachweise"/>
   <addaction name="menuExtras"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Hauptwerkzeugleiste</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actManageDatabase"/>
   <addaction name="actManageCertificate"/>
   <addaction name="actCreateSigningList"/>
   <addaction name="separator"/>
   <addaction name="actModeResult"/>
   <addaction name="actModeQualiMatrix"/>
   <addaction name="actModeTrainingData"/>
   <addaction name="separator"/>
   <addaction name="actSettings"/>
   <addaction name="separator"/>
   <addaction name="actAMS"/>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actManageDatabase">
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/opendatabase.svg</normaloff>:/icons/icons/opendatabase.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Datenbank verwalten</string>
   </property>
   <property name="toolTip">
    <string>Datenbank verwalten</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actCloseDatabase">
   <property name="text">
    <string>Datenbank &amp;schließen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>&amp;Über QualificationMatrix</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+ß</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/properties.svg</normaloff>:/icons/icons/properties.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Einstellungen</string>
   </property>
  </action>
  <action name="actAboutQt">
   <property name="text">
    <string>Über &amp;Qt</string>
   </property>
  </action>
  <action name="actCreateEmptyDatabase">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Leere Datebank erstellen</string>
   </property>
  </action>
  <action name="actSelectGerman">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="actSelectEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Englisch</string>
   </property>
  </action>
  <action name="actManageCertificate">
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/mail-attachment.svg</normaloff>:/icons/icons/mail-attachment.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Nachweise verwalten</string>
   </property>
   <property name="toolTip">
    <string>Nachweise verwalten</string>
   </property>
  </action>
  <action name="actIntegrityCheck">
   <property name="text">
    <string>&amp;Automatische Prüfung</string>
   </property>
  </action>
  <action name="actCreateSigningList">
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/traindata.svg</normaloff>:/icons/icons/traindata.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Unterschriftenliste erstellen</string>
   </property>
  </action>
  <action name="actModeResult">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/mail-mark-task.svg</normaloff>:/icons/icons/mail-mark-task.svg</iconset>
   </property>
   <property name="text">
    <string>Qualifizierungsergebnis</string>
   </property>
   <property name="toolTip">
    <string>Starte Modus Qualifizierungsergebnis</string>
   </property>
  </action>
  <action name="actModeQualiMatrix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/map-flat.svg</normaloff>:/icons/icons/map-flat.svg</iconset>
   </property>
   <property name="text">
    <string>Qualifzierungsmatrix</string>
   </property>
   <property name="toolTip">
    <string>Starte Modus Qualifizierungsmatrix</string>
   </property>
  </action>
  <action name="actModeTrainingData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/item.svg</normaloff>:/icons/icons/item.svg</iconset>
   </property>
   <property name="text">
    <string>Schulungsdaten</string>
   </property>
   <property name="toolTip">
    <string>Starte Modus Schulungsdaten</string>
   </property>
  </action>
  <action name="actLogin">
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Benutzer anmelden</string>
   </property>
  </action>
  <action name="actLogout">
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Aktuellen Benutzer abmelden</string>
   </property>
  </action>
  <action name="actAMS">
   <property name="icon">
    <iconset resource="../rcs/icons.qrc">
     <normaloff>:/icons/icons/im-user-offline.svg</normaloff>:/icons/icons/im-user-offline.svg</iconset>
   </property>
   <property name="text">
    <string>Benutzerverwaltung</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMInfoLabel</class>
   <extends>QLabel</extends>
   <header>framework/qminfolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rcs/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actAbout</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actManageDatabase</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>manageDatabase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actSettings</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>showAboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCloseDatabase</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>closeDatabase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actManageCertificate</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>manageCertificate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actIntegrityCheck</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>showCertificateIntegrityCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCreateSigningList</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>showCreateSigningList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actModeQualiMatrix</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>enterQualiMatrixMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actModeResult</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>enterResultMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actModeTrainingData</sender>
   <signal>triggered()</signal>
   <receiver>QMMainWindow</receiver>
   <slot>enterTrainingDataMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAbout()</slot>
  <slot>manageDatabase()</slot>
  <slot>showSettings()</slot>
  <slot>applyTrainDataFilter()</slot>
  <slot>updateResultModel()</slot>
  <slot>switchFuncFilter(bool)</slot>
  <slot>switchTrainFilter(bool)</slot>
  <slot>printToPDF()</slot>
  <slot>showAboutQt()</slot>
  <slot>closeDatabase()</slot>
  <slot>manageCertificate()</slot>
  <slot>showCertificateIntegrityCheck()</slot>
  <slot>showCreateSigningList()</slot>
  <slot>enterResultMode()</slot>
  <slot>enterQualiMatrixMode()</slot>
  <slot>enterTrainingDataMode()</slot>
  <slot>showAMSMenu()</slot>
 </slots>
</ui>
